src/calendar.rs

```diff
@@ -87,21 +87,15 @@ impl Calendar {
     }
 
     /// Prints to stdout
-    /// FIXME code repetition
     pub fn print(&self) -> Result<(), fmt::Error> {
-        let mut out = String::new();
-        self.fmt_write(&mut out)?;
-        print_crlf!("{}", out);
+        print_crlf!("{}", self);
         Ok(())
     }
```

Comment:
I actually thought we could just remove the `print()` entirely as that's now equivalent (modulo double crlf added by both fmt_write() and print() probably by mistake) to `print!("{}", calendar)` that everybody can call themselves.

API breakage, so this can probably be postponed. (take this as optional commend)

Reply:
the print is a bit special, since we need to take meticulous care to have the right lineendings apparently

Reply:
Makes sense.

Cool, nice little improvement this PR is.